test-it/testList.json

```diff
@@ -288,6 +288,7 @@
                 "TemplateBasics.via",
                 "TestVI_ArrayInsertPartial.via",
 		"TestVI_ArrayNonFlatInflate.via",
+		"TestVI_ArrayOfRefs.via",
```

Comment:
Do we need to fix a tab here? It seems the new line and the previous one are not aligned.

Reply:
Fixed.